cache.go

```diff
@@ -157,12 +157,12 @@ func (c *CacheLFUImmutable) ChannelUpdate(data []byte) (*ChannelUpdate, error) {
 	} else {
 		// unlikely
 		tmp := &Channel{}
-		if err := json.Unmarshal(data, channel); err != nil {
+		if err := json.Unmarshal(data, tmp); err != nil {
 			return nil, err
 		}
-		c.Patch(channel)
+		c.Patch(tmp)
 		channel = tmp.DeepCopy().(*Channel)
-		freshItem := c.Channels.CreateCacheableItem(tmp)
+		freshItem := c.Channels.CreateCacheableItem(channel)
```

Comment:
this should be tmp, otherwise the cache wont be read only

Reply:
Done.